### ISSUE TITLE
Increasing global timeouts?

### ISSUE THREAD

**Issue by joshuawhite929** [joshuawhite929/spmia_feedback#199]
John,

While testing Chapter 9 code, it was pretty common for me to receive a timeout on the very first service call.  When this was a read, I could easily re-run the call again to receive the result.  When running a delete however, that wasn't the case.  Though I would receive a timeout from Zuul, most times, the delete operation was successful.  Running it a second time didn't work as you would get an error.

Long story short... because we are not eager initializing these services, can we consider upping the default timeout so readers won't hit this problem?  Thoughts?

**Comment by carnellj** [comment]
Unfortunately its not something you can configure.  Eureka enforces a strict 30 second wait period on services. before they become available.  You can override it even in dev, but then if you are taking services in and out of your environment you can cause problems where a removed service is not unregistered immediately from Eureka.

Here is an interesting blog on this:  http://blog.abhijitsarkar.org/technical/netflix-eureka/

Its a bit of black magic
